administration/src/cards/SelfServiceCard.ts

```diff
@@ -1,10 +0,0 @@
-import { CardBlueprint } from './CardBlueprint'
-
-class SelfServiceCard extends CardBlueprint {
-  // Override isValid() to remove expiration check
-  isValid(): boolean {
-    return this.isFullNameValid() && this.extensions.every(ext => ext.isValid())
-  }
```

Comment:
Why?

Reply:
because it will be added in the backend. We don't have information in the frontend for that